cloudferrylib/os/context.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2016 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import contextlib
+import logging
+
+from cloudferrylib.os import clients
+from cloudferrylib.utils import utils
+from fabric import api as fab_api
```

Comment:
Organize your imports according to the [Import order template](http://docs.openstack.org/developer/hacking/#import-order-template)